x-pack/plugins/cloud_security_posture/server/create_indices/benchmark_score_mapping.ts

```diff
@@ -47,5 +47,27 @@ export const benchmarkScoreMapping: MappingTypeMapping = {
     low: {
       type: 'long',
     },
+    score_by_cloud_account: {
```

Comment:
remove the new mappings.
When we do not filter by it, a mapping is not needed.
Adding mapping comes with a cost. We should pay attention to which mappings we add